bundled/tool/lsp_server.py

```diff
@@ -215,6 +214,25 @@ def on_shutdown(_params: Optional[Any] = None) -> None:
     jsonrpc.shutdown_json_rpc()
 
 
+def _check_args() -> None:
```

Comment:
Is this basically checking if user provided args in their setting and giving them warning?

Reply:
IF they give args in their settings that we don't support or if it is structured in a way where it could be a mistake when user typed it.